llvm/test/Bitcode/local-type-scope.ll

```diff
@@ -0,0 +1,48 @@
+; RUN: llvm-as --disable-verify < %s -o %t0
+; RUN: opt --passes=verify %t0 -o /dev/null
+; RUN: llvm-dis %t0 -o - | FileCheck %s --implicit-check-not=DICompositeType
```

Comment:
This is a good test; but isn't the retainedNodes field going to be normalised by the first llvm-as, meaning that opt and llvm-dis only see a normalised input? If so, perhaps a round-trip test is sufficient, and something else is needed to exercise the metadata-loader for bitcode?